bindings.cpp

```diff
@@ -2546,7 +2546,7 @@ bool DeRestPluginPrivate::checkSensorBindingsForAttributeReporting(Sensor *senso
         sensor->modelId().startsWith(QLatin1String("Motion Sensor-A")) ||
         sensor->modelId().startsWith(QLatin1String("3300")) ||
         sensor->modelId().startsWith(QLatin1String("332")) ||
-        sensor->modelId().startsWith(QLatin1String("3200-S")) ||
+        sensor->modelId().startsWith(QLatin1String("3200-")) ||
```

Comment:
That's fine and I can definitively follow that. However, we should then mutually agree on some "coding guidelines" for the future to have that appropriately set into action. Especially since we are approaching a "transition time" to initiate required preparations.

Reply:
Absolutely agree, getting some Coding Guidelines would be helpful to streamline things in future.
I'll create a wiki page / issue for discussing this in the coming days as a starting point.